package-res/resources/web/prompting/builders/TextInputBuilder.js

```diff
@@ -15,16 +15,53 @@
  *
  */
 
+/**
+ * <h2>The Text Input Builder</h2>
+ *
+ * To use the TextInputBuilder you should require the appropriate file
+ * from Common-Ui:
+ *
+ * <pre><code>
+ *   require(['common-ui/builders/TextInputBuilder'], 
+ *     function(TextInputBuilder) { 
+ *       
+ *     }
+ *   );
+ * </code></pre>
+ * 
+ * To get the component you'll have to create a new instance of the builder and
+ * call the <code>build</code> method:
+ *
+ * <pre><code>
+ *   var textInputBuilder = new TextInputBuilder();
+ *
+ *   var textInputComponent = textInputBuilder.build(args);
+ * </code></pre>
+ *
+ * where 'args' is an object that contains the prompt panel and the parameters
+ * necessary for the component as per [the CDF documentation]{@link http://localhost:8080/pentaho/api/repos/:public:plugin-samples:pentaho-cdf:pentaho-cdf-require:30-documentation:30-component_reference:10-core:37-TextInputComponent:text_input_component.xcdf/generatedContent}.
```

Comment:
Do we not have place in the internet where our docs get published that we can link to rather than localhost? Chances are when someone sees the link and tries to follow it, they won't have the server running and or will not have the cdf doc plugin loaded in the platform.